docs/hosting-a-server-with-northstar/guides/pg9182-docker-installation-guide.md

```diff
@@ -1,97 +0,0 @@
----
```

Comment:
If you remove this page you also need to remove the entry from [`SUMMARY.md`](https://github.com/R2Northstar/NorthstarWiki/blob/5b873b0bc8362c83213019486f05d78857c5e825/docs/SUMMARY.md?plain=1#L38) ^^

Reply:
this is fair and I did not think about this